loomi/regulariza-diagonal-client
# Production Merge

## Production Merge

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description
